db/migrate/20210910163649_add_veteran_id_to_intakes.rb

```diff
@@ -0,0 +1,8 @@
+class AddVeteranIdToIntakes < Caseflow::Migration
+  disable_ddl_transaction!
+
+  def change
+    add_reference :intakes, :veteran, foreign_key: true, comment: "The ID of the veteran record associated with this intake", index: false
+    add_index :intakes, :veteran_id, algorithm: :concurrently
```

Comment:
Not needed in this case, but for consistency.
```suggestion
    add_safe_index :intakes, :veteran_id, algorithm: :concurrently
```